day03/ex09/ex09.md

```diff
@@ -2,14 +2,16 @@
 
 |                         |                         |
 | -----------------------:| ----------------------- |
-|   Turning directory :   |  ex09                   |
+|   Turn-in directory :   |  ex09                   |
 |   Files to turn in :    |  log_reg.py             |
-|   Forbidden function :  |  None                   |
+|   Forbidden functions : |  None                   |
 |   Remarks :             |  n/a                    |
+|   Ressources :          |   	Machine Learning Mooc / Week 3 - Regularized Logistic Regression: https://www.coursera.org/learn/machine-learning/lecture/4BHEy/regularized-logistic-regression | 
+
```

Comment:
resource/links format differs from previous exercices

Reply:
Hi! It's a good remark! Maybe `resources` would be more appropriate.

Reply:
Helpful links sounds better for me.
One should understand that the video is not compulsory and this is just a tiny help to dig in the right direction if it is needed.
folks who are working on the bootcamp must not understand that it is compulsory and the only ressource they should look at.